tests/Amadeus/ClientTest.php

```diff
@@ -168,14 +174,18 @@ public function testWillGetNullFromGetLastReqResHeadersWhenNoCallsWerMade()
 
     public function testCanDoDummyPnrRetrieveCall()
     {
-        $mockedSendResult = new Client\Session\Handler\SendResult();
+        $mockedSendResult              = new Client\Session\Handler\SendResult();
```

Comment:
Don't align the `=` with lines above/below, we don't do that anywhere in the library and I'd like to keep it that way.

Reply:
Please undo similar changes in other places.